aladdin/bash/container/create-namespace/create-namespace

```diff
@@ -2,7 +2,7 @@
 set -eu -o pipefail
 
 function create_namespace {
-    kubectl create namespace $1
+    kubectl create namespace $1 --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
I ran this and got:
> Warning: resource namespaces/default is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.


so it seems we're getting rid of one error but now we'll have this permanent warning?

Reply:
@jarojasm95 , i don't think so. During next runs, you shouldn't see this warning.